src/main/java/com/proofpoint/discovery/store/StoreResource.java

```diff
@@ -75,7 +81,7 @@ public Response setMultipleEntries(@PathParam("store") String storeName, List<En
 
     @GET
     @Produces({"application/x-jackson-smile", "application/json"})
-    public Response getAll(@PathParam("store") String storeName)
+    public Response getAll(@PathParam("store") String storeName, @Context HttpServletRequest request)
```

Comment:
Parameter not referenced/needed
```suggestion
    public Response getAll(@PathParam("store") String storeName)
```